scripts/analysis_0/nbconverted/2A_correct_variation_added.py

```diff
@@ -196,7 +200,7 @@
 print("Similarity between input vs permuted data is {}".format(permuted_score))
 
 
-# In[13]:
+# In[11]:
```

Comment:
If you don't need log values but only want it for log scale plots, you may want `scale_x_log` parameter in ggplot call instead.

Reply:
Very helpful tip.  For now, I am reverting back to plotting by experiment due to confusion about what "log number of experiments" means.  

I will continue to think about other possible display options for the x-axis to improve interpretation.  Please let me know if you have any suggestions regarding this.